src/components/QuestionInput.tsx

```diff
@@ -68,12 +75,62 @@ const ButtonSubmit = styled.button`
 `;
 
 const QuestionInput: React.FC = () => {
+  const {member: {isLogin}, setMember} = useContext(MemberContext);
   const [isFolded, setIsFolded] = useState<boolean>(true);
   const [title, setTitle] = useState<string>('');
```

Comment:
initialState로 type을 추론할 수 있지 않을까요? 
이런 경우엔 타입을 생략하는 걸 선호하는데, 어떻게 생각하시나요?

```suggestion
  const [isFolded, setIsFolded] = useState(true);
  const [title, setTitle] = useState('');
```

Reply:
일단 추론 가능한 경우에 type 생략을 선호하신다는 것에 충분히 공감합니다. 'type을 정해주어야한다'는 강박에 매몰되는 걸 경계해야겠구나 하고 생각했어요. 
그러면서 타입스크립트를 사용하는 이유에 대해 다시 한 번 생각해보았는데요, 예측 불가능한 실수를 미리 막는다는 점에서 이러한 경우에도 타입을 명시하는 게 더 좋지 않을까(제가 코드를 작성하며 어떤 실수를 할 지 모르니..) 하고 생각했어요. 또 레포를 몇 개 돌아다니면서 봤는데 명시해주는 경우가 더 많은 것 같더라고요. 그래서 그러한 경우에도 저는 타입을 명시해줄 것 같습니다!

Reply:
[제 pr에 남겨두셨던 코멘트](https://github.com/KTPT/Quna-client/pull/26#discussion_r564423906)와 비슷한 내용이군요.

어떻게하면 좋을지 생각해보죠!
저번에는 생략쪽에 기울었는데 다희님께서 다른 레포를 참고하시고 명시하는 경우가 더 많다니까 다시 고려해볼만 하네요.

Reply:
정답은 없는 부분이니 다음에 이야기 나눠보죠 🙂